src/addressResolvers/cache.ts

```diff
@@ -0,0 +1,42 @@
+import { Address, Handle } from './utils';
+import redis from '../helpers/redis';
+
+const KEY = 'address-resolvers';
+const TTL = 24 * 60 * 60 * 1e3;
```

Comment:
This is actually two years 😄 
<img width="467" alt="image" src="https://github.com/snapshot-labs/stamp/assets/15967809/7b50efb8-dc1f-4309-82bc-2104e7982c43">

Reply:
```suggestion
const TTL = 24 * 60 * 60;
```

Reply:
Actually there is a config file https://github.com/snapshot-labs/stamp/blob/add-addresses-resolvers/src/constants.json